umbraco-cloud/product-upgrades/major-upgrades.md

```diff
@@ -60,10 +60,14 @@ Look for the "**Upgrade from/to Umbraco xx"** boxes. These boxes contain importa
 
 ## Step 1: Enable .NET
 
+Before proceeding, you must determine whether the .NET framework version needs to be updated for your project. If no changes to the .NET version are required, you can skip this step and proceed with Step 2.
+
+Refer to the [Choose the correct .NET version](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading#choose-the-correct-.net-version) section to identify whether a .NET version update is necessary for your upgrade.
+
 1. Go to the project in the Umbraco Cloud portal.
 2. Navigate to **Configuration** -> **Advanced**.
 3. Scroll down to the **Runtime Settings** section.
-4. **Ensure that the latest version of .NET is enabled** for each environment on your Cloud project, by selecting it from the dropdown.
+4. Select the appropriate .NET version from the **Change .NET framework runtime for your Umbraco install** dropdown for each environment in your Cloud project.
```

Comment:
This actually shouldn't actually mention all environments - it should only be the first environment / development environment 🤔 

In a step 5, Step 1 is referenced, mentioning that you need to bump the .NET version on the next environment before deploying the upgrade to it. Hm!

Perhaps we could say:
"Select the correct .NET version from the [...] for the development environment, or the environment you are deploying the upgrade to."

Not sure if that becomes too confusing if you're only starting the upgrade 😅 
I think I definitely might be overcomplicating things here!!

Reply:
Umm, I don't think it matters cz we aren't providing separate steps for different environments. If I change it specifically to development I will have to add another line somewhere to repeat all these steps for another environment. In that case, it will just be making the guide too long. With this statement, there are chances the person doesn't miss the step for another environment 😅

As for step 5 list, the step to enable .net is mentioned twice in point 5 and point 11. Ideally, step 1 covers all versions, so those steps can be removed to reduce the number of steps while deploying the upgrade.

Reply:
Yeah, I did think about that as well.. But I'm not sure what impact changing this version actually has on the environment on Cloud. Like, would the environment continue to work even though the .NET version is changed without it being upgraded 🤔 I've asked the Cloud developers how this works .. if it doesn't impact the running environment, let's get rid of those extra steps in Step 5!

Reply:
OK, I finally got some answers.
Let's get the .NET version updated on all environments in Step 1, and then remove the references to this in Step 5 💪